imports/api/NewGameUtils.js

```diff
@@ -47,60 +47,58 @@ export var NewGameUtils = {
 
 		// This code could be streamlined, but let's leave it explicit
 		switch (gameType) {
-		case Constants.GAME_TYPE.HONG_KONG:
-			if (round <= 4)
-				return "東";
-			if (round > 4 && round <= 8)
-				return "南";
-			if (round > 8 && round <= 12)
-				return "西";
-			else //if (round > 12)
-				return "北";
-			break;
```

Comment:
Why are you so keen on putting breaks in every case?  I prefer to not include breaks in switch cases which are returned from.  Looks cleaner and the break is never called anyway.